### PR TITLE
docs: add core dump file docs

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR adds documentation for core dump file download. 

## Changed Pages

💻 [Add Preview URL for Page]()

## Jira Tickets

🎫 [PE-4733](https://spectrocloud.atlassian.net/issues/PE-4733)

## Backports

Can this PR be backported?

- [ ] No. 

[PE-4733]: https://spectrocloud.atlassian.net/browse/PE-4733?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## Prerequisites
+
+- You have configured your applications to direct core dump files to `/var/log/files` in the event of a crash. The steps
+  to do this vary by application, but the path must be exactly `/var/log/files`. This is the only path where files can
+  be downloaded from Local UI.
+
+- An operational Edge host installed in the `airgap` installation mode. For more information, refer to
+  [Installer Reference](../../edge-configuration/installer-reference.md).
+
+- Network access to your Edge host's Local UI port. The default port is 5800.
```
Comment (abhinavnagaraj) [inline docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md:22]: default port is 5080

Comment (lennessyy) [inline docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md:22]:
```suggestion
- Network access to your Edge host's Local UI port. The default port is 5080.
```

File: docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+sidebar_label: "Download Core Dump Files"
+title: "Download Core Dump Files"
+description: "Instructions for downloading core dump files from Local UI."
+hide_table_of_contents: false
+sidebar_position: 130
+tags: ["edge"]
+---
+
+If you have configured your applications to direct core dumps to the folder `/var/log/files` in the event of a crash,
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md:10]: I live for the phrase "core dumps".

File: docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+you can download the core dump files from Local UI for diagnostic purposes.
+
+## Prerequisites
+
+- You have configured your applications to direct core dump files to `/var/log/files` in the event of a crash. The steps
+  to do this vary by application, but the path must be exactly `/var/log/files`. This is the only path where files can
+  be downloaded from Local UI.
+
+- An operational Edge host installed in the `airgap` installation mode. For more information, refer to
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md:19]:
```suggestion
- An operational Edge host installed in the `airgap` mode. For more information, refer to
```

File: docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  [Installer Reference](../../edge-configuration/installer-reference.md).
+
+- Network access to your Edge host's Local UI port. The default port is 5080.
+
+## Procedure
+
+1. Log in to Local UI. For more information, refer to [Access Local UI Console](./access-console.md).
+
+2. From the left **Main Menu**, click on **Diagnostics**.
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md:28]:
```suggestion
2. From the left **Main Menu**, click **Diagnostics**.
```

File: docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+3. Click on the **Files** tab.
+
+4. Check the files you'd like to download.
+
+5. Click the **Download** button in the upper right corner.
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/download-core-dump.md:34]:
```suggestion
5. Click the **Download** button in the upper-right corner.
```

**Review by yuliiiah — APPROVED** [review]
Approved with tiny comments ✨

**Pull request merged**